### PR TITLE
samples: create topic model

### LINKED ISSUE #21: Add sample for creating a topic model
I'd like to add a sample for https://cloud.google.com/contact-center/insights/docs/topic-model#create_a_model. This sample will create a topic model in Insights.

### PR DESCRIPTION
**Pull request by TrucHLe**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-contact-center-insights/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #21  🦕

### CODE REVIEW

File: samples/snippets/create_issue_model.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+def create_issue_model(project_id: str) -> contact_center_insights_v1.IssueModel:
+    # Construct a parent resource.
+    parent = contact_center_insights_v1.ContactCenterInsightsClient.common_location_path(
+        project_id, "us-central1"
+    )
+
+    # Construct an issue model.
+    issue_model = contact_center_insights_v1.IssueModel()
+    issue_model.display_name = "my-model"
+    issue_model.input_data_config.filter = "medium=\"CHAT\""
```
Comment (busunkim96) [inline samples/snippets/create_issue_model.py:28]: Please make sure to have documentation "close" to this sample on cloud.google.com explaining the filter syntax.

File: samples/snippets/create_issue_model.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    # Call the Insights client to create an issue model.
+    insights_client = contact_center_insights_v1.ContactCenterInsightsClient()
+    issue_model_operation = insights_client.create_issue_model(
+        parent=parent, issue_model=issue_model
+    )
+
+    issue_model = issue_model_operation.result(timeout=600000)
```
Comment (busunkim96) [inline samples/snippets/create_issue_model.py:36]: Thanks for adding a timeout!  Is this the value you intended to add?  600,000 seconds is ~166 hours.

File: samples/snippets/test_create_issue_model.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    # See https://cloud.google.com/contact-center/insights/docs/topic-model.
+    list_request = contact_center_insights_v1.ListConversationsRequest()
+    list_request.page_size = 1000
+    list_request.parent = contact_center_insights_v1.ContactCenterInsightsClient.common_location_path(
+        project_id, "us-central1"
+    )
+
+    conversation_count = 0
+    while conversation_count < MIN_CONVERSATION_COUNT:
+        list_response = insights_client.list_conversations(request=list_request)
```
Comment (busunkim96) [inline samples/snippets/test_create_issue_model.py:49]: The generated docs don't do the best job of conveying this, but you don't need to handle the page token manually as the response object implements `__iter__` and will handle pagination for you. https://github.com/googleapis/python-contact-center-insights/blob/46b16129e08454fce9cf4ec9b2f7bc3a607807d0/google/cloud/contact_center_insights_v1/services/contact_center_insights/pagers.py#L31

Loop through conversations one by one:
```py
conversations = insights_client.list_conversations(request=list_request)

for conversation in conversations:
   # Do something with each item
```

Get a list of conversations:
 ```py
conversations = list(insights_client.list_conversations(request=list_request))

```

File: samples/snippets/create_issue_model.py (hunk 1 above)
Comment (TrucHLe) [reply samples/snippets/create_issue_model.py:28]: Ack! I created a bug for our team to document this value in the public doc.

File: samples/snippets/create_issue_model.py (hunk 2 above)
Comment (TrucHLe) [reply samples/snippets/create_issue_model.py:36]: Oops it should be 24 hours. I changed it back :)

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**